resources/pyenv_command.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+resource_name :pyenv_command
+provides :pyenv_command
+
+property :command, String, name_property: true
+property :user, String, required: true
+property :pyenv_path, String, required: true
+property :home_dir, String, default: lazy { ::File.expand_path("~#{user}") }
+
+default_action :run
+
+action :run do
+  cmd_env = {
+    'PYENV_ROOT' => new_resource.pyenv_path,
+    'PATH' => "#{new_resource.pyenv_path}/bin:#{ENV['PATH']}",
+    'USER' => new_resource.user,
+    'HOME' => new_resource.home_dir
+  }
+  bash "pyenv #{new_resource.command}" do
```

Comment:
Please move the Python Centos 6 to use this instead of pyenv cookbook

Reply:
I looked into this and it's more work than I thought it would be. Unless there's a compelling reason to invest the time I'd like to leave it as is and drop the centos6 case when that OS is deprecated.